#!/usr/bin/env python
PACKAGE = "cv_arm_detection"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("h_top", int_t, 0, "Low threshold for binary thresholding", 100, 0, 255)
gen.add("h_low", int_t, 0, "Low threshold for binary thresholding", 100, 0, 255)

gen.add("s_top", int_t, 0, "Low threshold for binary thresholding", 100, 0, 255)
gen.add("s_low", int_t, 0, "Low threshold for binary thresholding", 100, 0, 255)

gen.add("v_top", int_t, 0, "Low threshold for binary thresholding", 100, 0, 255)
gen.add("v_low", int_t, 0, "Low threshold for binary thresholding", 100, 0, 255)

gen.add("area_low", int_t, 0, "Low threshold for binary thresholding", 100, 0, 2000)
gen.add("area_high", int_t, 0, "Low threshold for binary thresholding", 1000, 0, 2000)

exit(gen.generate(PACKAGE, "arm_detection_node", "ArmDetection"))
